src/checks/evaluators.ts

```diff
@@ -49,3 +49,7 @@ export async function Evaluators(hook: IntraWebhook.Root, evaluations: Intra.Sca
 	}
 	return true;
 }
+
+export function random() {
+	return Math.random() < Config.randomEvalChance / 100;
+}
```

Comment:
Why ? The whole idea is to separate all potential checks into their own files to avoid cluttering them into a giant file like this.

Reply:
Also please start using docs.